.travis.yml

```diff
@@ -6,12 +6,19 @@ addons:
 
 rvm:
   - 2.4.0
+apt:
+  sources:
+    - ubuntu-sdk-team
+  packages:
+    - qt5-default
+    - libqt5webkit5-dev
+    - gstreamer1.0-plugins-base
+    - gstreamer1.0-tools
+    - gstreamer1.0-x
 
 install: bundle install --jobs=3 --retry=3 --with webkit
 
 env:
-  - SEXY_SETTINGS=driver=phantomjs
-  - SEXY_SETTINGS=driver=poltergeist
```

Comment:
Why poltergeist was removed?

Reply:
Phantom.js driver was removed from Selenium-Webdriver, but `poltergeist` is independent gem